modular_skyrat/modules/ERT_Factions/FTU/sound/placeholder.dm

```diff
@@ -0,0 +1 @@
+how he stand on 10 perc?
```

Comment:
removeeee

Reply:
There we go I'll just remove all the placeholder files